failsafe/circuit_breaker.py

```diff
@@ -25,30 +32,53 @@ def __init__(self, maximum_failures=2, reset_timeout_seconds=60):
         self.state = ClosedState(self)
 
     def allows_execution(self):
+        """
+        Returns a boolean indicating if the execution is allowed or not
+        depending on the state of the CircuitBreaker
+        """
         return self.state.allows_execution()
 
     def record_success(self):
+        """
+        Stores an execution success in the state of the CircuitBreaker
+        """
         self.state.record_success()
         logger.debug("Success recorded")
 
     def record_failure(self):
+        """
+        Stores an execution failure in the state of the CircuitBreaker
```

Comment:
Descriptions of `record_success` and `record_failure` functions say literally what code does, but I recon this isn't what user is most interested in.. I would suggest saying something similar as we say in `ClosedState` and `OpenState` classes. Explain what really will happen when those functions are called.